presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -11,11 +11,15 @@ import java.time.ZonedDateTime
 
 data class EventScreenState(
     val listOfPhotoUri: List<String> = mutableStateListOf<String>(),
-    val selectedVisitorType: VisitorFilterType = VisitorFilterType.ALL,
     val selectedVisitor: VisitorInfo? = null,
     val visitorEmail: String = "",
     val shouldShowVisitorDialog: Boolean = false,
+    val selectedVisitorFilterType: VisitorFilterType = VisitorFilterType.ALL,
+    val filteredVisitorsGoing: List<Attendee> = mutableStateListOf(),
+    val filteredVisitorsNotGoing: List<Attendee> = mutableStateListOf(),
     val isEmailVerified: Boolean = true,
+    val isAlreadyAdded: Boolean = false,
```

Comment:
Why do you need this?

Reply:
@MKiperszmid I'm checking if the user has already added a duplicate attendee

Reply:
But why do you need this as state? You can simply check this by searching for the added attendee ID in the existing attendees list